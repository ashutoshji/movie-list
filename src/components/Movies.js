import React from 'react';
import MovieCard from './MovieCard';
import PropTypes from 'prop-types';

const Movies = (props) => {
const movieWorld = props.movies
const rn=()=>{
    return Math.floor(Math.random() * 1000 + 1);
 }
return(movieWorld?(
  <ul className="movies">
    {props.movies.map(movie => (
      <li key={rn()}>
        <MovieCard {...movie} />
      </li>
    ))}
  </ul>
):"")};

Movies.propTypes = {
  movies: PropTypes.arrayOf(PropTypes.object)
};

export default Movies;