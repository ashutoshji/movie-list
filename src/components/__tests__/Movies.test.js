
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Movies from '../Movies';

const data = 

describe('Movies test', () => {
  it('Movies should match snapshot', () => {
    const component = renderer.create(<Movies
      movies={undefined} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
