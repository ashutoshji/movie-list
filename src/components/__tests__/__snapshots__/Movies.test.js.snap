// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movies test Movies should match snapshot 1`] = `""`;
